Source/Core/Core/IOS/Network/Socket.cpp

```diff
@@ -652,6 +652,11 @@ s32 WiiSockMan::AddSocket(s32 fd, bool is_rw)
   return wii_fd;
 }
 
+bool WiiSockMan::IsSocketBlocking(s32 wii_fd) const
+{
+  return WiiSockets.count(wii_fd) != 0 && !WiiSockets.at(wii_fd).nonBlock;
```

Comment:
Would a `find` be more performant?  I think having both of these calls does two lookups.